README.md

```diff
@@ -49,8 +49,8 @@ httpClient.get(url: "https://swift.org").whenComplete { result in
 }
 ```
 
-It is important to close client instance after use to cleanly shutdown underlying NIO ```EventLoopGroup```:
-```
+It is important to close the client instance, either in a ```defer``` statement or class' ```deinit``` after use to cleanly shutdown underlying NIO ```EventLoopGroup```:
```

Comment:
clean shutdown worth a bit more explanation and examples. i worry folks will read this as they are suppose to call `httpClient.syncShutdown()` on deinit

//cc @weissi

Reply:
I agree with @tomerd . Also please don't recommend or mention to use a class' `deinit`, that's poor style and will lead to really terrible bugs. People will still do it but at least we didn't recommend it :).

Reply:
😬great save!